web/styles/input_pill.css

```diff
@@ -60,6 +64,7 @@
         .pill {
             padding-right: 4px;
             cursor: not-allowed;
+            max-width: 100%;
```

Comment:
I don't think this and the below change is required. It should already be applied using the CSS added for `.pill` above.

Reply:
> I don't think this and the below change is required. It should already be applied using the CSS added for `.pill` above.

@sahil839 I've made the suggested changes.